### ISSUE TITLE
The Name property of a focusable element must not be null.

### ISSUE THREAD

**Issue by Karan4495** [dotnet/maui#6290]
GitHubTags:
#.NET MAUI; #A11ySev2; #A11yAuto; #AI4D; #A11yMAS; #A11yTCS; #Benchmark; #DesktopApp; #Win11; #WCAG4.2.1; #GH_.NETMAUI_Win32_Apr2022; #Object Information​;

### Description

The Name property of a focusable element must not be null.

**Application:** 
MauiApp8 has the following accessibility issue that needs investigation.

**Issue Details:**
Provide a UI Automation Name property that concisely identifies the element.

**Element path:** button ''

**To Reproduce and How to Fix guidance:** 
Please open the attached Accessibility Insights for Windows Test File (*.a11ytest) to reproduce the error in Accessibility Insights for Windows.

This accessibility issue was found using Accessibility Insights for Windows, a tool that helps debug and find accessibility issues earlier. Get more information & download this tool at https://aka.ms/AccessibilityInsights.

### Steps to Reproduce

1.Open .Net MAUI application solution Visual studio Enterprise 2022 Preview.
2.Tab till "MauiApp1" in solution explore and press enter.
3.Tab till Accept button and press enter.
4.Tab till Debug Target drop down to select Framework and then the net6.0-windows entry.
5.Tab till Windows Machine button and press enter.
6..Net MAUI template screen will open.
7.Tab till "Click me" button.
8.Start Accessibility insight for Desktop and see results.
![The Name property of a focusable element must not be null](https://user-images.githubusercontent.com/100186395/164169492-97a7b766-5dac-4b8c-9e9f-c5836366e03f.png)

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

Windows 11 Enterprise Version 21H2 OS build 22000.613

### Relevant log output

The Name property should be defined for all the focusable element

**Comment by rachelkang** [comment]
related fix: #6139

**Comment by Karan4495** [comment]
Verified issue in ".NET MAUI Template". issue is fixed so closing the bug
Windows 11 Enterprise Version 21H2 OS build 22000.675
Visual Studio Enterprise 2022 Int preview 17.3.0 Preview 2.0 [32513.392.main]
added attachment for references
![6290_Fixed](https://user-images.githubusercontent.com/100186395/168576490-730b62ed-c4fc-4247-8eb4-e7ca77009ff5.png)
